### PR TITLE
Fix get data from freed msg;

### PR DESCRIPTION
**Pull request by alvin1221**

### CODE REVIEW

**Review by JaylinYu — APPROVED** [review]

File: nanomq_cli/client.c [hunk 1]
```diff
@@ -1680,8 +1680,9 @@ static int
 quic_disconnect_cb(void *rmsg, void *arg)
 {
 	int reason = 0;
-	if (!rmsg) {
-		return 0;
+	if (msg == NULL) {
+		console("%s: Got null msg\n", __func__);
```
Comment (wanghaEMQ) [inline nanomq_cli/client.c:1684]: Plz add more details about the reason why a null msg be got.

File: nanomq_cli/client.c [hunk 2]
```diff
@@ -1744,6 +1745,10 @@ static int
 quic_msg_recv_cb(void *rmsg, void *arg)
 {
 	nng_msg *msg = rmsg;
+	if (msg == NULL) {
+		console("%s: Got null msg\n", __func__);
```
Comment (wanghaEMQ) [inline nanomq_cli/client.c:1749]: Plz add more details about the reason why a null msg be got.

**Review by lee-emqx — APPROVED** [review]

File: nanomq_cli/client.c (hunk 2 above)
Comment (alvin1221) [reply nanomq_cli/client.c:1749]: The max length of `nni_lmq` is enough, it's less likely to print this message...

Comment (wanghaEMQ) [reply nanomq_cli/client.c:1749]: Make the probable reasons printed on to the console

Comment (JaylinYu) [reply nanomq_cli/client.c:1749]: stand with wangha!

**Pull request merged**